server_environment/__manifest__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     "name": "server configuration environment files",
-    "version": "9.0.1.2.0",
+    "version": "10.0.1.2.0",
```

Comment:
I'm honestly not too sure the history on this one, being fairly newb to this community, but maybe it would be worth a mailing list discussion on the reasoning? 

I know it's in the contribution docs to do it this way, but I've honestly just chalked it up to someone with more experience than me determining it was better for some reason. Blindly following is probably not a good thing in hindsight though 😆